sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAdministrationConstants.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary>
+    /// Constant values for KeyVault Administration.
+    /// </summary>
+    public static class KeyVaultAdministrationConstants
```

Comment:
This should not be public. At best, it should be a public extensible enum (allows new values, if needed, without a new library) but see comment in the API file about helping users with scope.